en/drafts/originals/image-classification-neural-networks.md

```diff
@@ -25,150 +25,154 @@ doi: tbc
 
-What exactly is a convolution though? At its most basic, a convolution is a mathematical function resulting in two sets of information becoming converged. If you have used filters, such as blurs, in common image editing applications, you have used convolutions. Convolutions for images work by taking a filter, also known as a kernel, consisting of a grid of numbers, usually 3x3 or 5x5, and moving it over each pixel in the image. As the filter moves, the values in each overlapping pixel are multiplied by the values in the filter. Finally, the values for all the numbers in the grid are added together to create a single output.
+What exactly is a convolution though? At its most basic, a convolution is a mathematical function resulting in two sets of information becoming converged. If you have used filters, such as blurs, in common image editing applications, you have used convolutions. Convolutions for images work by taking a filter (also known as a kernel) consisting of a grid of numbers; usually 3x3 or 5x5, and moving it over each pixel in the image. As the filter moves, the values in each overlapping pixel are multiplied by the values in the filter. Finally, the values for all the numbers in the grid are added together to create a single output.
 
 Because the neural network takes the values from each grid and adds them together, the values given to the next layer are smaller than the original image. This new array of numbers is referred to as a "feature map" and makes training the neural network less computationally intensive. An activation function, such as ReLU, is also commonly used to transform all negative values to zero.
 
-Finally, a "pooling layer" is utilized. A pooling layer works similarly to a convolutional layer in that it takes a grid, usually 2x2, and passes it over each value in the feature map. In contrast with the convolution layer, however, the pooling layer simply takes the max or average value of the numbers in the grid. This creates a smaller feature map. Together, convolutions and pooling allow neural networks to do image classification even if the spatial arrangement of the pixels is different and without having to do as many calculations.
+Finally, a "pooling layer" is utilized. A pooling layer works similarly to a convolutional layer in that it takes a grid; usually 2x2, and passes it over each value in the feature map. In contrast with the convolution layer, however, the pooling layer simply takes the maximum or average value of the numbers in the grid. This creates a smaller feature map. Together, convolutions and pooling allow neural networks to perform image classification even if the spatial arrangement of the pixels is different and without having to do as many calculations.
```

Comment:
Q: maximum or average ? 

This is confusing to me. Which of these two values (the maximum or the average) ?

Reply:
It depends on the CNN. Some do maximum while others do averaging so the way it is written is correct.

Reply:
Thank you, @nabsiddiqui. I would suggest adding this in, within brackets perhaps. For example:

[...] the pooling layer simply takes the maximum or average value (depending on which CNN you are using) of the numbers in the grid [...]

I can apply this edit.

Reply:
Sure, if you want to add that, I am ok with that. Let me know if you need anything else on my end.